test/tdf-roundtrips.bats

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env bats
+
```

Comment:
Add description of process below.  Not long

Reply:
Better?